README.md

```diff
@@ -5,7 +5,8 @@ Moreover, Hoa aims at being a bridge between industrial and research worlds.
 
 # Hoa\Cli ![state](http://central.hoa-project.net/State/Cli)
 
-This meta-library provides the `hoa` command line.
+This meta-library provides the `hoa` command line. This is a shell tool to
+access to libraries' commands.
```

Comment:
"to access libraries' commands" ?

or

"to gain access to libraries' commands" ?